include/swift/Basic/InputFile.h

```diff
@@ -0,0 +1,66 @@
+//===--- InputFile.h --------------------------------------------*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
```

Comment:
2018, now!

Reply:
Yup. What about existing files? Should I update the dates in the future whenever I change one?